### ISSUE TITLE
About `operator-linebreak`'s default configuration.

### ISSUE THREAD

**Issue by mysticatea** [eslint/eslint#3604]
I was surprised.

```js
/* eslint operator-linebreak: 2 */
var foo = (
    a < 0 ? 0 :
    a > 9 ? 9 :
    /*else*/ a
);
```

This code is not warned by eslint, until v1.2.
But after I updated to v1.3, I see warning.

```
> eslint -v
v1.3.1
> eslint test.js --no-eslintrc

test.js
  3:16  error  ':' should be placed at the beginning of the line  operator-linebreak
  4:16  error  ':' should be placed at the beginning of the line  operator-linebreak

✖ 2 problems (2 errors, 0 warnings)
```

Until v1.2, this rule's default was `after`, wasn't it?

**Comment by BYK** [comment]
We added the following in #3512:

>The default configuration is to enforce line breaks after the operator except for the ternary operator ? and : following that.

This came from https://github.com/eslint/eslint/issues/3274#issuecomment-127799736 and https://github.com/eslint/eslint/issues/3274#issuecomment-128099785

If you set `"after"` explicitly, it would override these defaults. Please let us know if this is not okay so we can discuss more.

**Comment by mysticatea** [comment]
OK, I'll fix my config, thank you!
